cmd/fabric8-jenkins-proxy/main.go

```diff
@@ -152,24 +181,27 @@ func startWorkers(
 		}
 	}()
 
-	api := api.NewAPI(store)
+	apiService := goa.New("fabric8-jenkins-proxy-api")
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		srv := newAPIServer(api)
+		prepareAPIService(apiService)
 
 		go func() {
+			// Mount "stats" controller
+			c := api.NewStatsController(apiService, store)
+			app.MountStatsController(apiService, c)
+			router.CustomMuxHandle(apiService)
+
 			mainLogger.Infof("Starting API router on port %s", apiRouterPort)
-			listenAndServe(srv, cancel, config.GetHTTPSEnabled())
```

Comment:
since you removed the call to this function, shouldn't the function be deleted as well?